__fixtures__/test-project/web/src/components/AuthorCell/AuthorCell.stories.tsx

```diff
@@ -1,20 +1,22 @@
+import type { ComponentStory } from '@storybook/react'
+
 import { Loading, Empty, Failure, Success } from './AuthorCell'
 import { standard } from './AuthorCell.mock'
 
-export const loading = (args) => {
-  return Loading ? <Loading {...args} /> : null
+export const loading: ComponentStory<typeof Loading> = (args) => {
+  return Loading ? <Loading {...args} /> : <></>
```

Comment:
Just a stylistic preference, but I'd probably do this:

```suggestion
  return <Loading {...args} /> || <></>
```

Reply:
Looking at this again, do we actually need to handle this case? (Where `Loading` isn't exported by the Cell right?) If loading isn't defined, isn't TS going to complain when we import it furhter up?

```
import { Loading, Empty, Failure, Success } from './PostCell'
```

It looks like here we'll get something like `Module '"./PostCell"' has no exported member 'Loading'.ts(2305)`.

Reply:
That would be even better! Just make sure it works the same with strict mode on/off
And JS doesn't care, right? So for the JS template we'd still have to do something like this, yeah?